TempleApp/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,110 @@
     <string name="read_all">Read All</string>
     <string name="update">update</string>
     <string name="empty">"     "</string>
+    <string name="delete_donated">Selected To Delete Donated Money</string>
+    <string name="delete_pooja">Selected To Delete Registered Pooja</string>
+    <string name="REG">REG</string>
+    <string name="DON">DON</string>
+    <string name="wait">Hey Wait Please...</string>
+    <string name="deleting">Deleting...</string>
+    <string name="id_value">IDVALUE</string>
+    <string name="json_obj">Json obj</string>
+    <string name="result">result</string>
+    <string name="not_paid">NOT PAID</string>
+    <string name="DON">DON</string>
+    <string name="selected_donate">Selected To Donate Money</string>
+    <string name="REG">REG</string>
+    <string name="selected_register">Selected To Register New Pooja</string>
+    <string name="PAID">PAID</string>
+    <string name="NOT_PAID">NOT PAID</string>
+    <string name="valid_ID">Please enter a valid user ID</string>
+    <string name="wait">Hey Wait Please...</string>
+    <string name="insert">Inserting your values..</string>
+    <string name="json_obj">Json obj</string>
+    <string name="result">result</string>
+    <string name="exception">exception</string>
+    <string name="DON">DON</string>
+    <string name="UPDATE">Selected To update donated money details</string>
+    <string name="update_pooja">Selected To update Registered Pooja details</string>
+    <string name="PAID">PAID</string>
+    <string name="NOT_PAID">NOT PAID</string>
+    <string name="REG">REG</string>
+    <string name="WAIT">Hey Wait Please...</string>
+    <string name="JSON">I am getting your JSON</string>
+    <string name="json_obj">Json obj</string>
+    <string name="result">result</string>
+    <string name="selected_donate">Selected To Donate Money</string>
+    <string name="selected_pooja">Selected To Register New Pooja</string>
+    <string name="REG">REG</string>
+    <string name="DON">DON</string>
+    <string name="WAIT">Hey Wait Please...</string>
```

Comment:
Great work, but you given multiple times for same string like`DON` you have given more than 4 time please avoid that

Reply:
done

Reply:
Hey, still there are so many strings like for string `Update` you wrote two times `update` and `update_`  please make sure that for each string there will be only one resource